packages/api-graphql/src/GraphQLAPI.ts

```diff
@@ -35,6 +35,24 @@ export class GraphQLAPIClass extends InternalGraphQLAPIClass {
 	): Observable<GraphQLResult<T>> | Promise<GraphQLResult<T>> {
 		return super.graphql(options, additionalHeaders);
 	}
+
+	/**
+	 * Checks to see if an error thrown is from an api request cancellation
+	 * @param {any} error - Any error
+	 * @return {boolean} - A boolean indicating if the error was from an api request cancellation
+	 */
+	isCancelError(error: any): boolean {
+		return super.isCancelError(error);
+	}
+
+	/**
+	 * Cancels an inflight request. Only applicable for graphql queries and mutations
+	 * @param {any} request - request to cancel
+	 * @return {boolean} - A boolean indicating if the request was cancelled
```

Comment:
Nit: We use TSDoc instead of JSDoc in most of the places(I know we need to fix this in other places when we support API references)
```suggestion
	 * @param request - a promise of the request to cancel
	 * @returns A boolean indicating if the error was from an api request cancellation
```